afs/get_env.py

```diff
@@ -3,26 +3,18 @@
 import warnings
 import logging
 import afs.utils as utils
-import pkg_resources
-_logger = logging.getLogger(__name__)
-
+import afs
```

Comment:
Install a package named `advantech-afs` but use with another name `afs`? I think this might be a little confusing.

Reply:
Project renamed is to distinguish from the same name.
The name of import I have no idea strange or not.
For example, influxdb-python: https://www.influxdata.com/blog/getting-started-python-influxdb/
Due to some APP use advantech_afs, I am a little hesitant to change package name.